test/intest1.F90

```diff
@@ -25,6 +25,7 @@ program intest1
        bfmg(mxbf), cds3(mxds3)*6, tagpr*8, celem*60, cunit*22
   character*20 filnam / 'testfiles/IN_1' /
   character filost / 'r' /
+  integer i4dy, idxmsg
```

Comment:
The i4dy and idxmsg functions should be explicitly declared as integer*4, similar to the other library functions in line 16 which return integer values.

Remember the only library build now is with 4-byte integers, so all functions which return integer values will always return them as 4-byte integers.